src/App.js

```diff
@@ -44,8 +44,12 @@ const App = () => {
 				<Heading>Users</Heading>
 				<Users item={usersData} />
 			</Card>
-			{isValid && <EmptyValueModal onConfirm={errHandler} />}
-			{isValid && <NegValueModal onConfirm={errHandler} />}
+			{modalTypeToShow === "empty" && (
+				<EmptyValueModal onConfirm={errHandler} />
+			)}
+			{modalTypeToShow === "negative" && (
+				<NegValueModal onConfirm={errHandler} />
```

Comment:
i don't think there is a need of these two modal we can just disable the button if the user enters negative values or we can make it positive internally